kuka_rsi_hw_interface/include/kuka_rsi_hw_interface/rsi_command.h

```diff
@@ -64,7 +64,7 @@ class RSICommand
     el->SetAttribute("A4", std::to_string(joint_position_correction[3]));
     el->SetAttribute("A5", std::to_string(joint_position_correction[4]));
     el->SetAttribute("A6", std::to_string(joint_position_correction[5]));
-    if (!stop) {root->LinkEndChild(el);}
+    root->LinkEndChild(el);
 
     el = new TiXmlElement("Stop");
     el->LinkEndChild(new TiXmlText(std::to_string(static_cast<int>(stop))));
```

Comment:
will this leak if the TiXmlText's constructor throws? can it even happen?

Reply:
I think it cannot throw:
	
TiXmlText( const std::string& initValue ) : TiXmlNode (TiXmlNode::TINYXML_TEXT)
	{
		value = initValue;
		cdata = false;
	}